bump_version.exs

```diff
@@ -1,11 +1,61 @@
 #! /usr/bin/env elixir
 
-new_version =
-  "VERSION"
-  |> File.read!()
-  |> String.trim()
-  |> Version.parse!()
-  |> (fn ver -> "#{ver.major}.#{ver.minor}.#{ver.patch + 1}" end).()
-
-File.write("VERSION", new_version)
-IO.puts("Version bumped to #{new_version}")
+defmodule Git do
+  def latest_tag() do
+    "v" <> last_tag = execute(~w[tag])
+    |> String.split("\n")
+    |> List.last()
```

Comment:
I think this will give you the tag that is last alphabetically, not semantically. You know, `"10" < "9"` kind of stuff.

Reply:
```
git describe --tags --abbrev=0
```
^ that should always get you  the latest tag.

Reply:
humm! didn't realize that. I thought it always returns in order they were applied. alternatively I found this command
`git describe --tags $(git rev-list --tags --max-count=1)`` to return latest tag. This is returning the last tag as we need it. What do you think?

Reply:
> git describe --tags --abbrev=0

Waterpark used to use that, but it would occasionally return the second-latest tag, so we went brute force and implemented our own sort.

Reply:
@calvinb  yeah, FWIW, in rebar this is done in several steps, but that's for a very generic solution where by there's no control over tag schemes. I wonder how to replicate it getting the second tag vs the first?